src/styles/blocks/page.scss

```diff
@@ -0,0 +1,4 @@
+.page {
+  font-family: Roboto, sans-serif;
+  color: $main-text-color;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), you should use 2 spaces for indentation, but it seems like your indentation might be off. Please ensure you're using 2 spaces consistently throughout your SCSS files.